src/main/java/org/spongepowered/api/command/args/parsing/RawStringInputTokenizer.java

```diff
@@ -24,18 +24,16 @@
  */
 package org.spongepowered.api.command.args.parsing;
 
-import org.spongepowered.api.command.args.ArgumentParseException;
-
 import java.util.Collections;
 import java.util.List;
 
 class RawStringInputTokenizer implements InputTokenizer {
-    static final RawStringInputTokenizer INSTANCE = new RawStringInputTokenizer();
+    public static final RawStringInputTokenizer INSTANCE = new RawStringInputTokenizer();
```

Comment:
This was made public because both `QuotedStringsTokenzier` and `SpaceSplitInputTokenizer` have their instance marked as public. Regardless of which it is, they should be consistent. I'll leave that one to y'all to decide upon.

Reply:
Leave it as it was. You’re touching the API layer and we should be ensuring minimal, necessary, changes. There is no benefit to making this change, regardless of consistency.

Reply:
I'll revert it with the next commit, but I disagree with that statement.